prog/postgres/postgres_resource_nexus.rb

```diff
@@ -210,6 +210,8 @@ def create_certificate
     ).map(&:to_pem)
   end
 
+  @@dns_zone = nil
```

Comment:
I'm surprised that this works. So existence of the variable is enough and .freeze does not freeze the value. It kinda defeats the purpose of freezing, but still better than what we have.

Reply:
It's because the type may change but not the existence of the class variable. I actually wrote this as a prototype. Like you, I found it unsatisfying, because there is interesting dynamic behavior that could fail at run time while modifying the class. I'm not sure if I want to do go through with this hack.